docs/quickstart.qbk

```diff
@@ -0,0 +1,282 @@
+would like to try __hpx__ without a custom allocator at this point you can
+configure __hpx__ to use the system allocator in the next step.
+
+A full list of required and optional dependencies, including recommended
+versions is available at __prerequisites__.
+
+[heading Building HPX]
+
+Once you have the source code and the dependencies, set up a separate build
+directory and configure the project. Assuming all your dependencies are in paths
+known to __cmake__, the following gets you started:
+
+``
+    # In the HPX source directory
+    mkdir build && cd build
+    cmake -DCMAKE_INSTALL_PREFIX=/install/path ..
+    make install
+``
+
+This will build the core __hpx__ libraries and examples, and install them to
+your chosen location. If you want to install __hpx__ to system folders simply
+leave out the `CMAKE_INSTALL_PREFIX` option. This may take a while. To speed up
+the process launch more jobs by passing the `-jN` option to `make`.
+
+[tip Do not set only `-j` (i.e. `-j` without an explicit number of jobs) unless
+you have a lot of memory available on your machine. ]
+
+[tip If you want to change __cmake__ variables for your build it is usually a
+good idea to start with a clean build directory to avoid configuration problems.
+It is especially important that you use a clean build directory when changing
+between `Release` and `Debug` modes. ]
+
+If your dependencies are in custom locations you may need to tell __cmake__
+where to find them by passing one or more of the following options to __cmake__:
+
+``
+    -DBOOST_DIR=/path/to/boost
+    -DHWLOC_ROOT=/path/to/hwloc
+    -DTCMALLOC_ROOT=/path/to/tcmalloc
+    -DJEMALLOC_ROOT=/path/to/jemalloc
+``
+
+If you want to try __hpx__ without using a custom allocator pass
+`-DHPX_WITH_MALLOC=system` to __cmake__.
+
+[important If you are building __hpx__ for a system with more than 64 processing
+units you must change the __cmake__ variables `HPX_WITH_MORE_THAN_64_THREADS`
+(to `On`) and `HPX_WITH_MAX_CPU_COUNT` (to a value at least as big as the number
+of (virtual) cores on your system). ]
+
+To build the tests run `make tests`. To run the tests run either `make test` or
+use `ctest` for more control over which tests to run. You can run single tests
+for example with `ctest --output-on-failure -R
+tests.unit.parallel.algorithms.for_loop` or a whole group of tests with `ctest
+--output-on-failure -R tests.unit`.
+
+If you did not run `make install` earlier do so now or build the
+`simplest_hello_world_1` example by running
+
+``
+    make simplest_hello_world_1_exe
+``
+
+__hpx__ executables end up in the `bin` directory in your build directory. You
+can now run `simplest_hello_world` and should see the following output:
+
+``
+./bin/simplest_hello_world_1
+Hello World!
+``
+
+You've just run an example which prints `Hello World!` from the __hpx__ runtime.
+The source for the example is in `examples/quickstart/simplest_hello_world.cpp`.
```

Comment:
Shouldn't this be `simplest_hello_world1.cpp`?

Reply:
Yes, thanks!